rectangle.js

```diff
@@ -10,8 +10,13 @@ function diagonal(shape){
   return Math.sqrt(Math.pow(shape.width, 2) + Math.pow(shape.height, 2));
 };
 
+function isSquare(shape){
+  return (shape.height === shape.width);
+};
+
 module.exports = {
   area: area,
   perimeter: perimeter,
-  diagonal: diagonal
+  diagonal: diagonal,
+  isSquare: isSquare
 }
```

Comment:
With ES6, syntactic sugar was built in for objects like this, where the key and value share the same name:

 module.exports = {
    area,
    perimeter,
   diagonal,
   isSquare
  }

This is commonly used for module.exports.